### ISSUE TITLE
Why can't it start

### ISSUE THREAD

**Issue by YJZyjz185672** [aviddot/Pancakeswap-v2-trading-bot#16]
![image](https://user-images.githubusercontent.com/44958443/119078757-653f3a00-ba29-11eb-8e45-923abe929485.png)

**Comment by aviddot** [comment]
Problem with your address/pk you have tried to insert in the configfile.py

**Comment by rigservice** [comment]
Yeah I got it tht figured out but now I can't even get that far lol where
should I go extract cups today and load onuwhat program like a Brower or
the should open on the thr own sorry ik I'm difficult
